skimage/measure/_moments.py

```diff
@@ -235,7 +235,9 @@ def centroid(image):
         The centroid of the (nonzero) pixels in ``image``.
     """
     M = moments_central(image, center=(0,) * image.ndim, order=1)
-    center = M[tuple(np.eye(image.ndim, dtype=int))]
+    center = (M[tuple(np.eye(image.ndim, dtype=int))]  # array of weighted sums
+                                                       # for each axis
+              / M[(0,) * image.ndim])  # weighted sum of all points
```

Comment:
Whoopsies! Thanks for spotting this @kne42! Yikes!

Could you fix the formatting to satisfy PEP8? (Operators should be at the end of the preceding line.)

```python
# weighted sums along each axis divided by weighted sum of all points
center = (M[tuple(np.eye(image.ndim, dtype=int))] /
          M[(0,) * image.ndim])
```

Reply:
See #2816

Reply:
@kne42 I'll be damned. Did PEP8 get updated or did I misread like 5y ago and have been doing it wrong ever since???

Reply:
@jni Nah, whoever added code W503 to PEP8 probably just didn’t read the [PEP8 write-up](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator) or the write-up was modified later but the code was not.